src/query/catalog/src/statistics/data_cache_statistics.rs

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 Datafuse Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::atomic::AtomicUsize;
+
+use serde::Deserialize;
+use serde::Serialize;
+
+#[derive(Default)]
+pub struct DataCacheMetrics {
```

Comment:
Make these fileds name also same as struct of `DataCacheMetricValues`?

Reply:
:ok_hand: